x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/action/CrossClustersQueryIT.java

```diff
@@ -121,17 +142,23 @@ public void testMetadataIndex() {
         }
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/106273")
```

Comment:
The motivation of this PR is to address this test failure. By using a preference, we can ensure that we execute against the same sets of data nodes in that test.